packages/lesswrong/lib/generated/fragmentTypes.d.ts

```diff
@@ -1676,17 +1676,11 @@ interface ChildRelatedPostRelList { // fragment on PostRelations
 }
 
 interface SuggestAlignmentComment extends CommentsList { // fragment on Comments
-  readonly post: SuggestAlignmentComment_post|null,
+  readonly post: PostsMinimumInfo|null,
```

Comment:
This seems likely to be an accident left from some other branch.

Reply:
Yeah. Do I need to do anything about it?

Reply:
Yeah, you should regenerate the current types, by running `yarn run generate` from a console, in parallel to a running instance